src/drive/lib/migration/qualification.js

```diff
@@ -100,15 +100,15 @@ export const extractFilesToMigrate = files => {
  * We changed some labels set by cozy-scanner: this method
  * transform them with the new one.
  *
- * @param {string} label - The old qualification label
+ * @param {string} oldLabel - The old qualification label
  * @returns {string} The new qualification label
  */
 const getNewLabelSetFromCozyScanner = oldLabel => {
   if (oldLabel === 'registration') {
     return 'vehicle_registration'
   }
   if (oldLabel === 'insurance_card') {
-    return 'national_insurance_card'
+    return 'national_health_insurance_card'
```

Comment:
If a PR depends on another one, can you change the target branch for this one? In order to not review the same thing. Once the previous will be merged, the target branche will be automatically set to master here.